src/PlayerHitBillboardPhysicsComponent.cpp

```diff
@@ -0,0 +1,19 @@
+#include "PlayerHitBillboardPhysicsComponent.h"
+
+#include "GameManager.h"
+#include "GameObject.h"
+#include "GameWorld.h"
+
+void PlayerHitBillboardPhysicsComponent::updatePhysics(float deltaTime) {
+	currentLifeTime += deltaTime;
+
+	if (currentLifeTime > maxLifeTime) {
+		GameManager& gameManager = GameManager::instance();
+		GameWorld& world = gameManager.getGameWorld();
+
+		world.rmDynamicGameObject(holder_);
+	}
+	else {
+		holder_->setScale(glm::vec3(1.0) + glm::vec3(sin(currentLifeTime * M_PI)));
```

Comment:
This should throw an error. A temp variable will have to be created and assigned to the function parameter `glm::vec3& newScale` that will hold a reference to a temp. I'll build when I get home and see if this is a prob but I think it will be.